docs/src/app/components/pages/components/GridList/Page.jsx

```diff
@@ -11,17 +11,31 @@ import GridListExampleSimple from './ExampleSimple';
 import gridListExampleComplexCode from '!raw!./ExampleComplex';
 import GridListExampleComplex from './ExampleComplex';
 
+const descriptions = {
+  simple: 'A simple example of a scrollable `GridList`.',
+  complex: 'This example demonstrates "featured" tiles, using the `rows` and `cols` props to adjust the size of the ' +
+  'tile. The tiles have a customised title, positioned at the top and with a custom gradient `titleBackground`.',
+};
+
 const GridListPage = () => (
   <div>
     <MarkdownElement text={gridListReadmeText} />
-    <CodeExample code={gridListExampleSimpleCode}>
+    <CodeExample
+      title="Simple Grid List"
+      description={descriptions.simple}
+      code={gridListExampleSimpleCode}
+    >
       <GridListExampleSimple />
     </CodeExample>
-    <CodeExample code={gridListExampleComplexCode} >
+    <CodeExample
+      title="Complex Grid List"
+      description={descriptions.complex}
+      code={gridListExampleComplexCode}
+    >
       <GridListExampleComplex />
     </CodeExample>
-    <PropTypeDescription code={gridListCode}/>
-    <PropTypeDescription code={gridTileCode}/>
+    <PropTypeDescription header="### Grid List Properties" code={gridListCode}/>
+    <PropTypeDescription header="### Grid Tile Properties" code={gridTileCode}/>
```

Comment:
Yes, I was undecided, since we use the feature name in some places, and the component name in others. It would be good to have clarity, and therefore consistency in when to use one or the other. 

Also, for features with only one component, should it be "SomeComponent Properties", or just "Properties". Obviously where there are sub components, disbiguation is essential.

Similarly, should the example titles, for example: "Controlled example", "Controlled Left Nav", or "Controlled LeftNav".